compileopts/config.go

```diff
@@ -462,6 +462,15 @@ func (c *Config) WasmAbi() string {
 	return c.Target.WasmAbi
 }
 
+// Emulator returns the emulator target config
+func (c *Config) Emulator() []string {
+	var emulator []string
+	for _, s := range c.Target.Emulator {
+		emulator = append(emulator, strings.ReplaceAll(s, "{root}", goenv.Get("TINYGOROOT")))
+	}
```

Comment:
This works, but a more efficient version is the following:

```suggestion
	emulator = make([]string, len(c.Target.Emulator))
	for i, s := range c.Target.Emulator {
		emulator[i] = strings.ReplaceAll(s, "{root}", goenv.Get("TINYGOROOT"))
	}
```

...probably not worth the effort though. Just an idea.